packages/react-bindings/src/hooks/useAccessibility.ts

```diff
@@ -62,8 +70,30 @@ const useAccessibility = <Props>(
     actionHandlers,
   )
 
-  return <SlotProps extends Record<string, any>>(slotName: string, slotProps: SlotProps) =>
+  const getA11Props: UseAccessibilityResult = (slotName, slotProps) =>
     mergeProps(slotName, slotProps, definition)
+
+  // Provides an experimental handling for FocusZone definition in behaviors
+  getA11Props.unstable_withFocusZone = (children: React.ReactElement) => {
+    if (definition.focusZone) {
+      let element: React.ReactElement = children
+
+      if (process.env.NODE_ENV !== 'production') {
```

Comment:
It's already restricted by typings, but it will be also good to have a runtime checks